scripts/gceworker.sh

```diff
@@ -24,6 +24,21 @@ function start_and_wait() {
     retry gcloud compute ssh "${1}" --command=true || true
 }
 
+function refresh_ssh_config() {
+    IP=$($0 ip)
+    if ! grep -q "${FQNAME}" ~/.ssh/config; then
+      echo "No alias found for ${FQNAME} in ~/.ssh/config. Creating one now with the instance external ip."
+      echo "Host ${FQNAME}
+  HostName ${IP}
+  IdentityFile $HOME/.ssh/google_compute_engine
+  UserKnownHostsFile=$HOME/.ssh/google_compute_known_hosts
+  IdentitiesOnly=yes
+  CheckHostIP=no" >> ~/.ssh/config
```

Comment:
Editing someone's top level .ssh/config, particularly without asking , feels a little presumptive. I wonder if we could instead edit a `.ssh/gceworker_config` and then tell the user that they can `Include` directive that file if they are so inclined?

I don't know about others but I keep all my core dotfiles source control and anything that just messes with them like this is a source of friction.